addons/web/static/src/webclient/webclient.js

```diff
@@ -100,6 +100,7 @@ export class WebClient extends Component {
         // we do not want any other listener to execute.
         if (
             ev.ctrlKey &&
+            !(ev.target.contentEditable === 'true') &&
```

Comment:
This does not look like the right property to test, in fact. This is not easy to check on this particular property because it can have multiple values, see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/contentEditable and you don't want to compute the meaning of `inherit`. However, you can use `isContentEditable` instead, see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/isContentEditable

Reply:
This is the property I've been looking for :dizzy: